app/domain/operations/fdsh/evidence_verification_request.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+
+module Operations
+  module Fdsh
+    # This class is responsible for validating an application object and constructing a payload entity for FDSH service.
+    class EvidenceVerificationRequest
+      include Dry::Monads[:result, :do]
+
+      def call(evidence, event)
```

Comment:
Below is the standard pattern for all operations in call method.
```suggestion
def call(evidence, event)
   application = yield find_application()
   response = yield send_fdsh_hub_call()
   result = yield some_name()
   Success()
```

can you modify the logic to use the above pattern?